wp-content/themes/openlab/lib/sidebar-funcs.php

```diff
@@ -281,7 +278,7 @@ function openlab_members_sidebar_blocks($mobile_hide = false) {
             <ul class="sidebar-sublinks portfolio-sublinks inline-element-list">
 
                 <li class="portfolio-site-link bold">
-                    <a class="bold no-deco" href="<?php openlab_user_portfolio_url() ?>">
+                    <a class="bold no-deco test" href="<?php openlab_user_portfolio_url() ?>">
```

Comment:
This `test` class is left over from debugging, right?

Reply:
Removed :)